gradio/components/plot.py

```diff
@@ -56,6 +57,7 @@ def __init__(
         """
         Parameters:
             value: Optionally, supply a default plot object to display, must be a matplotlib, plotly, altair, or bokeh figure, or a callable. If callable, the function will be called whenever the app loads to set the initial value of the component.
+            format: File format to save matplotlib plots as, such as 'jpg' or 'png'. If set to None, will default to 'png'.
```

Comment:
```suggestion
            format: File format in which to send plot data to the frontend, such as 'jpg' or 'png'. If set to None, will default to 'png'.
```

Reply:
This only applies to Matplotlib plots so I'll keep that part and make the other suggestions in wording